### PR TITLE
[W5.11][W13-4]Kenneth Chua

### PR DESCRIPTION
**Pull request by kennethcsj**

### CODE REVIEW

**Comment by tonykoh** [comment]
I can see that you made sure to commit after every implementation. Good job!

**Review by guiyong96 — COMMENTED** [review]
Clear sortcommandtest which is easy to understand

**Review by Sri-vatsa — APPROVED** [review]
Good job on implementing the sort command and updating tests 👍

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (Sri-vatsa) [inline src/seedu/addressbook/parser/Parser.java:14]: Try not to import all. It is always good practice to import only the classes that you need.

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 2]
```diff
@@ -21,6 +21,7 @@ public CommandResult execute() {
                 + "\n" + ClearCommand.MESSAGE_USAGE
                 + "\n" + FindCommand.MESSAGE_USAGE
                 + "\n" + ListCommand.MESSAGE_USAGE
+                + "\n" + SortCommand.MESSAGE_USAGE
```
Comment (Sri-vatsa) [inline src/seedu/addressbook/commands/HelpCommand.java:24]: Great that you have updated the help command! :)

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorting all persons: 'sort'
```
Comment (Sri-vatsa) [inline docs/UserGuide.adoc:75]: Good job on updating the user guide as well!

File: test/input.txt [hunk 4]
```diff
@@ -86,6 +86,13 @@
   viewall 4
   viewall 5
 
+##########################################################
+# test sort command
```
Comment (Sri-vatsa) [inline test/input.txt]: Yepp, adding a test case to ensure your method works is a good practice :)

**Review by Sri-vatsa — APPROVED** [review]
Great work! Review my comments and close the PR.

**Pull request closed without merging**